From af5f6a28f7671ac14861802c73359c642463a410 Mon Sep 17 00:00:00 2001
From: Lukas Bulwahn <lukas.bulwahn@bmw-carit.de>
Date: Sat, 29 Apr 2017 19:38:47 +0200
Subject: [PATCH] do not require python2

When only a python3 interpreter is available, CMake fails with:

```
| CMake Error at /home/lukas/work/repositories/openembedded.org/openembedded-core/build/tmp-glibc/work/core2-64-oe-linux/ros-mavlink/git-r0/recipe-sysroot-native/usr/share/cmake-3.7/Modules/FindPackageHandleStandardArgs.cmake:138 (message):
|   Could NOT find PythonInterp (missing: PYTHON_EXECUTABLE) (Required is at
|   least version "2")
| Call Stack (most recent call first):
|   /home/lukas/work/repositories/openembedded.org/openembedded-core/build/tmp-glibc/work/core2-64-oe-linux/ros-mavlink/git-r0/recipe-sysroot-native/usr/share/cmake-3.7/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
|   /home/lukas/work/repositories/openembedded.org/openembedded-core/build/tmp-glibc/work/core2-64-oe-linux/ros-mavlink/git-r0/recipe-sysroot-native/usr/share/cmake-3.7/Modules/FindPythonInterp.cmake:151 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
|   CMakeLists.txt:22 (find_package)
|
|
| -- Configuring incomplete, errors occurred!
```

After dropping this one line, CMake does not insist on python2,
and configures mavlink successfully.

Signed-off-by: Lukas Bulwahn <lukas.bulwahn@gmail.com>
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 594ecb3..26a4210 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,7 +18,6 @@ include(GNUInstallDirs)
 set(CMAKE_INSTALL_LIBDIR lib)
 
 # find libraries with cmake modules
-set(PythonInterp_FIND_VERSION "2")
 find_package(PythonInterp REQUIRED)
 
 # Try to read package version from package.xml
-- 
1.9.1

